Sources/SwiftUICharts/Helpers.swift

```diff
@@ -211,6 +211,10 @@ public class ChartData: ObservableObject, Identifiable {
     public func onlyPoints() -> [Double] {
         return self.points.map{ $0.1 }
     }
+    
+    public func getData() -> [(String,Double)] {
```

Comment:
I think it might help to name these values in the tuple. Like: `(x: String, y: Double)` or maybe `(key: String, value: Double)`?

Reply:
Added in the names in the latest commit. Merging in now.